qa/tasks/barbican.py

```diff
@@ -4,8 +4,8 @@
 import argparse
 import contextlib
 import logging
-import httplib
-from urlparse import urlparse
+from six.moves import http_client
+from six.moves.urllib.parse import urlparse
```

Comment:
Since we already changed this in many places. Could you please use teuthology.util.compat for url parse and urllib staff?

Reply:
+1 for using six.moves.urllib.parse. one wrapper module is enough, imo

Reply:
@sebastian-philipp well, teuthology is the one module, the six, is extra here ;-)

Reply:
@kshtsk if it's wrong or cause more efforts in future, i am all for changing it, otherwise i'd prefer keeping as it is. because i think this code will be removed along with `teuthology.util.compat` once py3 migration is finished, i don't really want to take extra efforts to rework it.

Reply:
I'm fine with both. Given that we use both already. So keeping six here might be the simplest solution

Reply:
Honestly, I would prefer to get rid of six at all, and use home-brew solution for each case. So we are not dependent on third-party modules.

Reply:
Really?
```
$ cat requirements2.txt | wc -l
112 (icnluding some comments/empty lines)
```

So you care about a single module which is well tested and really often used?

Reply:
"Well tested" are you kidding? What is well tested in python?

Reply:
https://pypistats.org/top